.rubocop.yml

```diff
@@ -5,9 +5,42 @@ AllCops:
     - 'lib/rubygems/resolver/molinillo/**/*'
   TargetRubyVersion: 2.2
 
+Layout/AccessModifierIndentation:
+  Enabled: true
+
+Layout/BlockAlignment:
+  Enabled: true
+
+Layout/CaseIndentation:
+  Enabled: true
+
+Layout/ClosingParenthesisIndentation:
+  Enabled: true
+
 Layout/CommentIndentation:
   Enabled: true
 
+Layout/ElseAlignment:
+  Enabled: true
+
+# Force Unix line endings.
+Layout/EndOfLine:
+  Enabled: true
+  EnforcedStyle: lf
+
+Layout/IndentHash:
+  Enabled: true
+  EnforcedStyle: consistent
+
+# NOTE: When used with --auto-correct, this only seems to auto-correct the
+#       first array item for some reason.
+Layout/IndentArray:
+  Enabled: true
+  EnforcedStyle: consistent
+  Exclude:
+    # This is a generated file.
+    - lib/rubygems/util/licenses.rb
```

Comment:
I think we can fix this at the source by removing spaces from [here](https://github.com/rubygems/rubygems/blob/cc881a806afc90731d5a912415aa945726978b54/util/generate_spdx_license_list.rb#L28) and [here](https://github.com/rubygems/rubygems/blob/cc881a806afc90731d5a912415aa945726978b54/util/generate_spdx_license_list.rb#L33).

Reply:
Given the size of this PR, I'd like to make a second follow-up PR to address this one after this gets merged, if that's okay with other folks.

Reply:
Sounds good to me! :+1: